cumulusci/tests/pytest_plugins/pytest_sf_orgconnect.py

```diff
@@ -24,15 +26,19 @@ def project_config(runtime):
 
 
 @pytest.fixture(scope="session")
-def org_config(request, runtime):
+def org_config(request, runtime, fallback_orgconfig):
     """Get an org config with an active access token.
 
     Specify the org name using the --org option when running pytest.
-    Or else it will use your default CCI org.
+    Or else it will use a dummy org.
     """
-    org_name = request.config.getoption("--org")
-    org_name, org_config = runtime.get_org(org_name)
-    org_config.refresh_oauth_token(runtime.keychain)
+    org_name = sf_pytest_orgname(request)
+    if org_name:
+        org_name, org_config = runtime.get_org(org_name)
+        org_config.refresh_oauth_token(runtime.keychain)
+    else:
+        return fallback_orgconfig()
```

Comment:
Maybe worth a comment about how fallback_orgconfig is defined differently for integration tests?

Reply:
https://github.com/SFDO-Tooling/CumulusCI/pull/1816/commits/e1bdf07bc3899f6d19b51a13bf5c997503dc9c1d#diff-20c920c476d998d34927d826ade54c1dR41